File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/cpp/ex07_contador_de_numeros.cpp
/*
    Autor: Kael Elodie Whitmore
    Data: Junho/2026
    Descrição: Lê 10 números e conta quantos são positivos, negativos e zeros.
*/

#include <iostream>
using namespace std;

int main()
{
    int numero;
    int contador = 1;
    int positivos = 0;
    int negativos = 0;
    int zeros = 0;

    while (contador <= 10)
    {
        cout << "Digite o " << contador << "o numero: ";
        cin >> numero;

        if (numero > 0)
        {
            positivos++;
        }
        else if (numero < 0)
        {
            negativos++;
        }
        else
        {
            zeros++;
        }

        contador++;
    }

    cout << "Positivos: " << positivos << endl;
    cout << "Negativos: " << negativos << endl;
    cout << "Zeros: " << zeros << endl;

    return 0;
}

File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/python/ex07_contador_de_numeros.py
# Autor: Matheus F. Mation
# Data: Agosto/2026 
# Descrição: Lê 10 números e conta quantos são positivos, negativos e zeros.

numero = 0
contador = 1
positivos = 0
negativos = 0
zeros = 0

while contador <= 10:
    print(f"Digite o {contador}° numero")
    numero = int(input("numero:"))
    if numero > 0:
        positivos += 1
    
    elif numero < 0:
        negativos += 1     

    else:
        zeros += 1
    contador += 1

print(f"Positivos: {positivos}")
print(f"Negativos: {negativos}")
print(f"Zeros: {zeros}")